src/lib.rs

```diff
@@ -143,8 +143,10 @@ mod status {
 #[cfg(target_os = "linux")]
```

Comment:
I must have run cargo fmt. This might need to be reverted.

Reply:
Yeah, let's revert that, and maybe bring it in a separate PR if you think it's really needed.